wp_api/src/lib.rs

```diff
@@ -183,20 +184,27 @@ impl TryFrom<BoolOrString> for WpResponseString {
     }
 }
 
-#[derive(Debug, Serialize, Deserialize, uniffi::Enum)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, uniffi::Enum)]
 #[serde(untagged)]
 pub enum BoolOrString {
     Bool(bool),
     String(String),
 }
 
-#[derive(Debug, Serialize, Deserialize, uniffi::Enum)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, uniffi::Enum)]
 #[serde(untagged)]
 pub enum BoolOrVecString {
     Bool(bool),
     VecString(Vec<String>),
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, uniffi::Enum)]
+#[serde(untagged)]
+pub enum IntegerOrString {
+    Integer(i64),
+    String(String),
+}
```

Comment:
Any reason not using the existing `DeserializeI64OrStringVisitor`?

Reply:
`DeserializeI64OrStringVisitor` is used to parse the data into `i64` whereas here we want to be able to support both `i64` and an arbitrary string (not just a number). Furthermore, this type is directly exposed to the clients, it's not an intermediate type that's only used for parsing.

Reply:
I wonder if it'd be valuable to introduce a `Either<A, B>` type, which can represent all the `FooOrBar` types above.

Reply:
`uniffi` does not support generics, so that doesn't help us in this case.